docs/architecture/adr-008-blocksize-independent-commitment.md

```diff
@@ -0,0 +1,100 @@
+# ADR 008: Blocksize independent Commitments for Messages
+
+## Changelog
+
+- 03.08.2022: Initial Draft
+
+## Context
+
+Currently, commitments in Celestia are dependent on the size of the block. The following diagram visualizes how this can happen. The visualizations in this ADR are without parity shares/erasure encoding.
```

Comment:
just to note that removing the neccessity to malleate transactions depending on square size is a massive amount of work, considering that this is a major responsibility of the application. That's **not** to say that we shouldn't!! this is purely a comment

Reply:
Should i add this as a neutral consequence, because it will take a lot of work but could simplify the design. ?